SPECS/mysql/mysql.spec

```diff
@@ -25,14 +24,13 @@ Requires:       %{name} = %{version}-%{release}
 %description devel
 Development headers for developing applications linking to maridb
 
-
 %prep
 %setup -q %{name}-boost-%{version}
 
 %build
 cmake . \
-      -DCMAKE_INSTALL_PREFIX=/usr   \
-      -DWITH_BOOST=boost/boost_1_72_0 \
+      -DCMAKE_INSTALL_PREFIX=%{_prefix}   \
+      -DWITH_BOOST=boost/boost_1_73_0 \
```

Comment:
>boost_1_73_0 [](start = 25, length = 12)

Why do we need this version change?

Reply:
MySQL includes the boost source in its tarball, so each time it releases a new tarball it uses a different version of boost. `Source0` name includes `boost` because of that.

Reply:
Thanks for the explanation.

---
In reply to: [517691456](https://github.com/microsoft/CBL-Mariner/pull/341#discussion_r517691456) [](ancestors = 517691456)